Re-exported `scale-ts` types from `@polkadot-api/substrate-bindings` are not found with `"moduleResolution": "bundler"`

For example, adding `@polkadot-api/substrate-bindings` and creating a `test.ts` file with
```ts
import { Decoder } from "@polkadot-api/substrate-bindings"

declare const x: Decoder
```

And using a `tsconfig.json` with `"moduleResolution": "bundler"`

Outputs the following error
```
➜  polkadot-api git:(dfcee8e) ✗ cd examples/extension-dapp 
➜  extension-dapp git:(dfcee8e) ✗ pnpm build

> extension-dapp@0.0.0 build /Users/kratico/ghq/github.com/paritytech/polkadot-api/examples/extension-dapp
> tsc && vite build

src/test.ts:1:10 - error TS2459: Module '"@polkadot-api/substrate-bindings"' declares 'Decoder' locally, but it is not exported.

1 import { Decoder } from "@polkadot-api/substrate-bindings"
           ~~~~~~~

  ../../packages/substrate-bindings/dist/index.d.mts:2:26
    2 import { Codec, Encoder, Decoder, CodecType } from 'scale-ts';
                               ~~~~~~~
    'Decoder' is declared here.


Found 1 error in src/test.ts:1
```

<img width="778" alt="image" src="https://github.com/paritytech/polkadot-api/assets/1209171/9dadbff1-6763-4d0d-9f32-67e521b91969">

Then, the `substrate-bindings/dist/index.d.mts` file from `@polkadot-api/substrate-bindings` starts with
```ts
import * as scale_ts from 'scale-ts';
import { Codec, Encoder, Decoder, CodecType } from 'scale-ts';
export * from 'scale-ts';
```

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -47,7 +47,7 @@
     "ora": "^7.0.1",
     "read-pkg": "^8.1.0",
     "rxjs": "^7.8.1",
-    "scale-ts": "^1.4.0",
+    "scale-ts": "^1.4.2",
     "tsc-prog": "^2.3.0",
     "type-fest": "^4.3.1",
     "typescript": "^5.1.6",

--- a/packages/substrate-bindings/package.json
+++ b/packages/substrate-bindings/package.json
@@ -50,7 +50,7 @@
     "@noble/hashes": "^1.3.1",
     "@scure/base": "^1.1.1",
     "@polkadot-api/utils": "workspace:*",
-    "scale-ts": "^1.4.0"
+    "scale-ts": "^1.4.2"
   },
   "devDependencies": {
     "bigint-conversion": "^2.4.1",

